apps/extension-e2e/src/classes/browser-context.class.ts

```diff
@@ -7,11 +7,13 @@ export const DEFAULT_SEED_PHRASE = getEnv('DEFAULT_SEED_PHRASE');
 export const DEFAULT_PASSWORD = getEnv('DEFAULT_PASSWORD');
 export const DEFAULT_HD_ACCOUNT_PRIVATE_KEY = getEnv('DEFAULT_HD_ACCOUNT_PRIVATE_KEY');
 export const SEED_PHRASE_FOR_IMPORT = getEnv('SEED_PHRASE_FOR_IMPORT');
+export const PRIVATE_KEY_FOR_IMPORT = getEnv('PRIVATE_KEY_FOR_IMPORT');
 
 export class BrowserContext {
   public static browser: Browser;
   public static page: Page;
   public static recorder: PuppeteerScreenRecorder;
   public static seedPhrase = DEFAULT_SEED_PHRASE;
   public static password = DEFAULT_PASSWORD;
+  public static privateKey = PRIVATE_KEY_FOR_IMPORT;
```

Comment:
no need to store this value in `BrowserContext`  
just directly use `PRIVATE_KEY_FOR_IMPORT`